### PR TITLE
Move Logo moved to top left of application bar

### PR DESCRIPTION
**Pull request by osaosula**
Logo has been placed in the application bar by:
1. Removing it from main menu
2. Putting it as the 1st menu item
3. Removing the position prop from the Logo Box

### CODE REVIEW

File: src/MenuItem.jsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import { Drawer, List, ListItem, Link } from "@mui/material";
 import { Link as RouterLink } from "react-router-dom";
+import Logo from "./Logo";
 function MenuItem() {
   return (
     <Drawer anchor="left" open={true} variant="persistent">
       <List>
+        <ListItem>
+          <Logo />
```
Comment (BorisLaboratory) [inline src/MenuItem.jsx:9]: I'm not sure if logo should be a list item. 
1. Criteria say logo should be inside app bar
2. Recall that the other user story handling the Drawer is required to now separate  the Drawer from the app bar. So the Drawer will be under the app bar , I believe

File: src/MenuItem.jsx [hunk 2]
```diff
@@ -30,7 +34,7 @@ function MenuItem() {
           >
             Admin
           </Link>
-        </ListItem>
+        </ListItem>{" "}
```
Comment (BorisLaboratory) [inline src/MenuItem.jsx:37]: Redundant comment

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]

File: src/MenuItem.jsx (hunk 1 above)
Comment (osaosula) [reply src/MenuItem.jsx:9]: In that case, the Logo was never the problem it was directly in the application bar, the list menu should be properly placed, then the logo would show. Should I revert to the former? Then  the adjustment can be made to the menu.

Comment (BorisLaboratory) [reply src/MenuItem.jsx:9]: It looks like you are right .
During the meeting I thought I heard Delvin say the logo was  good to go. Confirm from Delvin. May be very minor adjustment

Comment (dellydel) [reply src/MenuItem.jsx:9]: The logo should be inside of the app bar. The issue that I was seeing is that the drawer component is overlapping the app bar. So I think repositioning the drawer component to be below the app bar is the solution

Comment (osaosula) [reply src/MenuItem.jsx]: I have adjusted the Drawer to accommodate the Logo

**Pull request closed without merging**